lazy_tensor_core/ts_native_functions.yaml

```diff
@@ -2,10 +2,12 @@ backend: Lazy
 cpp_namespace: torch_lazy_tensors
 full_codegen:
   - _log_softmax
-  - mean
   - addcdiv
   - addcmul
   - cos
+  - gelu
+  - gelu_backward
+  - mean
```

Comment:
alphabetized? nice. was thinking about whether we can lint that someday.. :)

Reply:
Yea, I didn't bother to learn linter for this small patch.